integration/spark/databricks/README.md

```diff
@@ -0,0 +1,52 @@
+# Open Lineage in Databricks
+
+Open Lineage's [Spark Integration](../README.md) can be installed on Databricks leveraging init scripts. Please note, Databricks on Google Cloud does not currently support the DBFS CLI so the proposed solution will not work for Google Cloud until that feature is enabled. 
+
+* [Azure Databricks Init Scripts](https://docs.microsoft.com/en-us/azure/databricks/clusters/init-scripts)
+* [GCP Databricks Init Scripts](https://docs.gcp.databricks.com/clusters/init-scripts.html)
+* [AWS Databricks Init Scripts](https://docs.databricks.com/clusters/init-scripts.html)
+
+## Databricks Install Instructions
+
+Follow the steps below to enable Open Lineage on Databricks.
+
+* Build the jar via Gradle
```

Comment:
Maybe provide a link to the latest built jar in maven central? https://search.maven.org/remote_content?g=io.openlineage&a=openlineage-spark&v=LATEST